src/providers/uploadsManager/uploadsManager.tsx

```diff
@@ -36,6 +52,31 @@ export const UploadsManager: React.FC = () => {
     },
   })
 
+  // display snackbar when video upload is complete
+  useEffect(() => {
```

Comment:
Could you explain a bit on what basis the snackbar is shown? I don't fully get the logic atm

Reply:
I was looking for an easy way to determine which uploads are finished. So I keep all channel's uploads in ref and every time it changes, I check if it's status changed. If it changed to `completed` I display snackbar. I hope I explained it clearly 😉